nixos/modules/services/home-automation/matter-server.nix

```diff
@@ -89,7 +89,7 @@ in
         AmbientCapabilities = "";
         CapabilityBoundingSet = "";
         DevicePolicy = "closed";
-        DynamicUser = true;
+        DynamicUser = false;
```

Comment:
```suggestion
```
as which user does this run? I couldn't find a thing

Reply:
Indeed, no user means it runs as root.

Reply:
I believe this is now resolved. We're back to using DynamicUser=true, and instead of TemporaryFileSystem=/ we're using an ephemeral root in the systemd-managed runtime directory.